### PR TITLE
feat: monthly good optional choice exp false door lightbox

### PR DESCRIPTION
**Pull request by egoroza**
- implements false door experience/lightbox for monthly good optional choice experiment
- also updates `max-width` for mg optional choice landing page to be wider per design request

mobile:
Screen Shot 2022-04-22 at 12 01 35 AM

desktop:
Screen Shot 2022-04-22 at 12 01 44 AM

### CODE REVIEW

File: src/pages/MonthlyGood/MonthlyGoodLandingPage.vue [hunk 1]
```diff
@@ -195,6 +196,41 @@
...
+		<!-- Monthly Good False Door Exp Optional Choice -->
+		<kv-lightbox
+			title="Thanks for your interest!"
+			:visible="showActiveLightbox"
+			@lightbox-closed="closeLightbox"
+		>
+			<div class="tw-text-center tw-flex tw-flex-col tw-gap-3 tw-my-8 lg:tw-w-8/12 md:tw-w-full">
+				<!-- eslint-disable-next-line max-len -->
+				<p>Choosing a loan in Monthly Good is not yet available. If you want to choose a loan each month, you may like auto-deposit.</p>
+				<div class="tw-flex tw-flex-col tw-gap-2 lg:tw-w-10/12 md:tw-w-full tw-self-center">
```
Comment (mcstover) [inline src/pages/MonthlyGood/MonthlyGoodLandingPage.vue:209]: These buttons look really wide on desktop... maybe we need a max width of  300px or so added as a style attribute...

Comment (egoroza) [reply src/pages/MonthlyGood/MonthlyGoodLandingPage.vue:209]: oops - grabbed the wrong screencap but yeah it's still a little wide. can set that 👍

**Comment by roger-in-kiva** [comment]
Content for mobile is aligned at the middle of the screen in figma, having a gap on the top and rounded corners.

**Comment by roger-in-kiva** [comment]
Maybe decreasing the font size for the title, we could have a better view for ipad...

**Comment by roger-in-kiva** [comment]
We need to update this copy, Susan is dummy data
Screen Shot 2022-04-20 at 13 36 52

**Comment by egoroza** [comment]
> We need to update this copy, Susan is dummy data Screen Shot 2022-04-20 at 13 36 52

This supposed to be part of the image preview and is not actually functional according to product/design, i.e. just an image example and not implemented in the code as of now

**Comment by egoroza** [comment]
> Maybe decreasing the font size for the title, we could have a better view for ipad...

I'm not sure if it makes sense to have a custom title for this lightbox since it would be good to have parity between the other `kv-lightbox` components across the site but happy to change that if others feel strongly about this

**Comment by mcstover** [comment]
We should use the standard lightbox title and sizing for this. No need to alter type sizes per viewport as those changes are already built into the lightbox component.

File: src/pages/MonthlyGood/MonthlyGoodLandingPage.vue [hunk 2]
```diff
@@ -210,11 +246,13 @@ import WwwPage from '@/components/WwwFrame/WwwPage';
 
 import KvHero from '@/components/Kv/KvHero';
 import KvContentfulImg from '@/components/Kv/KvContentfulImg';
+import KvLightbox from '@/components/Kv/KvLightbox';
```
[the remark this answers is not in the document]
Comment (mcstover) [reply src/pages/MonthlyGood/MonthlyGoodLandingPage.vue:249]: @egoroza please use the new version of our lightbox.

If you need to style the header differently than the default you can pass in a named slot with your special tailwind classes.

[the remark this answers is not in the document]
Comment (mcstover) [reply src/pages/MonthlyGood/MonthlyGoodLandingPage.vue:249]: There's an example of a custom header within our new lightbox here: https://github.com/kiva/ui/blob/main/src/components/BorrowerProfile/LendersAndTeams.vue

One notable change you'll see is that the mobile version will be more like an overlay card with rounded corners at the top.

File: src/pages/MonthlyGood/MonthlyGoodLandingPage.vue [hunk 3]
```diff
@@ -195,6 +196,43 @@
 				:questions="frequentlyAskedQuestions"
 			/>
 		</div>
+
+		<!-- Monthly Good False Door Exp Optional Choice -->
+		<kv-lightbox
+			:visible="showActiveLightbox"
+			@lightbox-closed="closeLightbox"
+		>
+			<h2 class="tw-text-h2 tw-text-center">
```
Comment (egoroza) [inline src/pages/MonthlyGood/MonthlyGoodLandingPage.vue:205]: design wants this centered and not in line with the close icon, looks like putting this in `title` or `<template #header>` as a custom header keeps this in the same row as the close icon

**Review by mcstover — APPROVED** [review]

**Pull request merged**